doc/introduction/basic_reference.md

```diff
@@ -182,8 +182,7 @@ The math module is the backbone of Mr Mustard, which consists in the [Math](http
 Mr Mustard comes with a plug-and-play backends through a math interface. You can use it as a drop-in replacement for tensorflow or pytorch and your code will be plug-and-play too!
 ```python
 from mrmustard import settings
-from mrmustard.math import Math
-math = Math()
+import mrmustard.math as math
```

Comment:
can you also `from mrmustard import math`?

Reply:
Not with thsi PR, as it gives error: `ImportError: cannot import name 'math' from '<unknown module name>' (unknown location)`
But after this goes in, i would like to move a few objects to namespace, and this will become possible

Reply:
ah, because you're playing with the namespace and the modules